### PR TITLE
ci: create issue mark duplicate

### PR DESCRIPTION
**Pull request by xrkffgg**
### User facing changelog

None

#### 1. mark duplicate

ref: https://github.com/cypress-io/cypress/issues/14282 https://github.com/cypress-io/cypress/issues/9530 https://github.com/cypress-io/cypress/issues/8933

Now when you type `Duplicate of #xxx`, the GitHub Actions will help you add then `type: duplicate` label auto.

At the same time, it will close the issue if you just click `Comment`. And will remove the `stage: awaiting response` label if the issue has. This `remove-labels` support multiple, but I don't know what you need, you can tell me.

#### 2. issue labeled reply

ref: https://github.com/cypress-io/cypress/issues/14405

If you add `stage: needs information` label, the GitHub Actions will help you comment this.

@jennifer-shehane 

### How has the user experience changed?
None

### PR Tasks

- [x] Have tests been added/updated?
- [x] Has the original issue or this PR been tagged with a release in ZenHub? 
- [x] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [x] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [x] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: .github/workflows/issue-labeled.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+on:
+  issues:
+    types: [labeled]
+
+jobs:
+  issue-reply:
+    runs-on: ubuntu-latest
+    steps:
+      - name: help wanted
```
Comment (xrkffgg) [inline .github/workflows/issue-labeled.yml:11]:
```suggestion
      - name: needs information
```

**Review by jennifer-shehane — CHANGES_REQUESTED** [review]
@bahmutov thoughts? 

I don't think that I'd like the GitHub action to auto-comment. I don't always want to comment at the time of adding the 'needs:information' label. I usually type in some more specific info in my comment that I'd also like. Also, not sure how the comment shows up, it shows as commenting by GitHub Action? 

The duplicate action seems fine though.

**Comment by xrkffgg** [comment]
You can see in https://github.com/xrkffgg/test-ci/issues/48. 

image

This is default token. And if you set another. You can see https://github.com/ant-design/ant-design/issues/28767

image

**Comment by xrkffgg** [comment]
> I don't always want to comment at the time of adding the 'needs:information' label.

If so, do I need to remove this? `issue labeled reply`

**Review by jennifer-shehane — DISMISSED** [review]
Yes, I would remove the `issue labeled reply` part of the action. 

We do have a cypress-bot we use for other messages. 

Screen Shot 2021-01-11 at 2 20 05 PM

**Comment by xrkffgg** [comment]
Okay, I have removed it.

File: .github/workflows/issue-mark-duplicate.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  mark-duplicate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: mark-duplicate
+        uses: actions-cool/issues-helper@v1.8
```
Comment (xrkffgg) [inline .github/workflows/issue-mark-duplicate.yml:12]:
```suggestion
        uses: actions-cool/issues-helper@1.x
```

**Comment by jennifer-shehane** [comment]
@xrkffgg Thanks for the PR and suggestions. Unfortunately our team has decided against pulling in a 3rd party GitHub Action into our repo at this time. I'll be closing this PR.

**Comment by xrkffgg** [comment]
OK~

**Pull request closed without merging**